harvard-lts/mps-viewer
# Help link plugin + refined resource linking

**Help link plugin + refined resource linking**
* * *

**JIRA Ticket**: 
* [LTSVIEWER-120: Update URN with human readable link](https://jira.huit.harvard.edu/browse/LTSVIEWER-120)
* [LTSVIEWER-63: Add link to help](https://jira.huit.harvard.edu/browse/LTSVIEWER-63)

# What does this Pull Request do?
* Adds a dropdown dialog menu for users to get help
* Changed copyright URN into plain language
* Small miscellaneous UI updates

# How should this be tested?
Rebuild container from this branch, navigate to any item page, and check for the following UI updates:
* Image tools are now hidden by default (used to have a floating icon in the top right)
* Click on the three dots button in the top right
    * Menu items have been reordered and new item for help has been added
        * Old order: hide image tools, share, download
        * New order: download, share, show image tools, help
    * Click on the help menu item to see a dialog box with three choices (all three of these link off to external help options)
* Open the sidebar on the left side of the screen
    * Click on the copyright "c" icon...the url underneath "License" has been replaced with plain language link that says "Copyright & Terms of Use"; link still brings you to the same place
    * Click on the icon below copyright...the title of this section has been changed from "Index" to "Table of Contents"
* Harvard Library logo in bottom left of viewer now links back to library.harvard instead of digital collections
* Link to a version 3 dev manifest has been included below the viewer so that a user could add this resource for comparison
* Footer copyright year has been updated to "2022"

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no

# Interested parties
@phil-plencner-hl @enriquediaz
